README.md

```diff
@@ -75,14 +75,14 @@ npm install
 npm test
 ```
 
-More extensive tests for this package are contained in the [tei-publisher-app](https://github.com/eeditiones/tei-publisher-app/tree/feature/open-api/test) repository.
+More extensive tests for this package are contained in the [tei-publisher-app](https://github.com/eeditiones/tei-publisher-app/tree/master) repository.
```

Comment:
Consider dropping the `/tree/master` bit, and let GitHub take folks to the default branch at: https://github.com/eeditiones/tei-publisher-app.

Reply:
Well, the link should go to the tests of tei-publisher-app so it definitely has to change to be meaningful. And for this to work I believe we have to link to a specific branch at least. But I might be wrong. 
More importantly the number of tests in the Roaster repository has grown significantly since this paragraph was added to the readme. We could it altogether, as it lowers confidence in the library that it is not tested well enough.

Reply:
I rephrased the paragraph slightly and the link now points to the location @joewiz suggested.